dashboard/src/components/Catalog/Catalog.test.tsx

```diff
@@ -366,7 +366,7 @@ describe("pagination and chart fetching", () => {
       />,
     );
     expect(wrapper.find("CatalogItems").prop("page")).toBe(1);
-    expect(wrapper.find("CatalogItems>CatalogItem>ChartCatalogItem").length).toBe(0);
+    expect(wrapper.find("ChartCatalogItem").length).toBe(0);
```

Comment:
btw, just fyi in case you don't know, the difference between using `wrapper.find(Component)` and `wrapper.find("Component")` is that the second verifies that there is a component named  "Component" while the first ensures that the exact component (the one you are importing in the test file) is present.

You don't need to change this, it's just that I usually do the former since it's more accurate, in case there are two components that are named the same.

Reply:
Ahhh, good to know! Thanks for the tip!